<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private '.service_locator.HtctPft' shared service.

return $this->privates['.service_locator.HtctPft'] = new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($this->getService, [
    'cpt' => ['privates', '.errored..service_locator.HtctPft.App\\Entity\\Compte', NULL, 'Cannot autowire service ".service_locator.HtctPft": it references class "App\\Entity\\Compte" but no such service exists.'],
    'dpt' => ['privates', '.errored..service_locator.HtctPft.App\\Entity\\Depot', NULL, 'Cannot autowire service ".service_locator.HtctPft": it references class "App\\Entity\\Depot" but no such service exists.'],
    'violations' => ['privates', '.errored..service_locator.HtctPft.Symfony\\Component\\Validator\\ConstraintViolationList', NULL, 'Cannot autowire service ".service_locator.HtctPft": it references class "Symfony\\Component\\Validator\\ConstraintViolationList" but no such service exists.'],
], [
    'cpt' => 'App\\Entity\\Compte',
    'dpt' => 'App\\Entity\\Depot',
    'violations' => 'Symfony\\Component\\Validator\\ConstraintViolationList',
]);
